tibo-mllr/activeviam-projet
# Summary page improvement

Not related to an issue

# Description:

Adding pie charts on the summary page for Elapsed timings (not the execution context ones) and the retrievals per type
- Color is the same for a given retrievalType on all the page
- retrievals are now ranked in descending order for their elapsed timings and their number of appearances

# How to test:

Launch the app
go to localhost:3000/summary
Check that there is two pie charts working properly, with the correct color correspondance and data ordered in descending order

# Other notes:
